go/vt/vttablet/tabletmanager/vreplication/table_plan_builder.go

```diff
@@ -102,22 +113,24 @@ func buildPlayerPlan(filter *binlogdatapb.Filter, tableKeys map[string][]string)
 				}
 				plan.VStreamFilter.Rules = append(plan.VStreamFilter.Rules, sendRule)
 				tablePlan := &TablePlan{
-					Name:     tableName,
-					SendRule: sendRule,
+					TargetName: tableName,
+					SendRule:   sendRule,
 				}
 				plan.TargetTables[tableName] = tablePlan
 				plan.TablePlans[tableName] = tablePlan
+				continue nextTable
```

Comment:
Was not having this line of code a bug/inefficiency in the previous version of code?